api/src/main/java/com/codeforcommunity/rest/subrouter/ProtectedSiteRouter.java

```diff
@@ -416,4 +417,16 @@ private void handleApproveSiteImage(RoutingContext ctx) {
 
     end(ctx.response(), 200);
   }
+  private void registerGetUnapprovedImages(Router router) {
+    Route unapprovedSiteImages = router.get("/unapproved_images");
+    unapprovedSiteImages.handler(this::handleGetUnapprovedImages);
+  }
+
+  private void handleGetUnapprovedImages(RoutingContext ctx) {
+    JWTData userData = ctx.get("jwt_data");
+    List<SiteEntryImage> images = processor.getUnapprovedImages(userData);
+    end(ctx.response(), 200,
+            JsonObject.mapFrom(Collections.singletonMap("images", images))
```

Comment:
Just clarifying question - does this return a list of base64 string images?

Reply:
It gets sent in this format which doesn't have base64 encoding but instead a link to s3 image url: 
{
  "images": [
    {
      "imageId": INT,
      "uploaderUsername": STRING,
      "uploadedAt": TIMESTAMP,
      "imageUrl": STRING
    },
    ...
  ]  
}